specs/separate_configuration_files_per_component.adoc

```diff
@@ -0,0 +1,246 @@
+  configuration.
+* Add new python file to keep configuration details.
+* Modify the config file to read the configuration details from particular
+  local config file.
+* Modify manager to read and store it own configuration into etcd under cluster
+  specific hierarchy in the central store.
+* Create new flow and atoms to allow modification on configuration in etcd. And
+  it should affect local configuration file also.
+
+=== Tendrl/ceph_integration impact
+
+* Modify the installation document to copy configuration file with proper name.
+* Rename sample configuration file
+* Modify the spec file to copy configuration file with proper name.
+* Modify the tendrl definition file with new object, atoms, flows, pre-run,
+  post-run, attributes, type, uuid, version, inputs, description for the
+  configuration.
+* Add new python file to keep configuration details.
+* Modify the config file to read the configuration details from particular
+  local config file.
+* Modify manager to read and store it own configuration into etcd under cluster
+  specific hierarchy in the central store.
+* Create new flow and atoms to allow modification on configuration in etcd. And
+  it should affect local configuration file also.
+
+=== Tendrl/performance_monitoring impact
+
+* Modify the installation document to copy configuration file with proper name.
+* Rename sample configuration file
+* Modify the spec file to copy configuration file with proper name.
+* Modify the yaml file with new object, atoms, flows, pre-run,
+  post-run, attributes, type, uuid, version, inputs, description for the
+  configuration.
+* Add new python file to keep configuration details.
+* Modify the config file to read the configuration details from particular
+  local config file.
+* Modify manager to read and store it own configuration into etcd with proper
+  key.
+* Create new flow and atoms to allow modification on configuration in etcd. And
+  it should affect local configuration file also.
+
+=== Tendrl/alerting impact
+
+* Modify the installation document to copy configuration file with proper name.
+* Rename sample configuration file
+* Modify the spec file to copy configuration file with proper name.
+* Modify the yaml file with new object, atoms, flows, pre-run,
+  post-run, attributes, type, uuid, version, inputs, description for the
+  configuration.
+* Add new python file to keep configuration details.
+* Modify the config file to read the configuration details from particular
+  local config file.
+* Modify manager to read and store it own configuration into etcd with proper
+  key.
+* Create new flow and atoms to allow modification on configuration in etcd. And
+  it should affect local configuration file also.
+
+
+== Security impact
+
+None
+
+== Notifications/Monitoring impact
+
+None
+
+== Other end user impact
+
+None
+
+== Performance Impact
+
+None
+
+== Other deployer impact
+
+None
+
+== Developer impact
+
+None
+
+== Implementation
+
```

Comment:
Here you should add only the specs issue URL and rest should be added in "References" section